debian/postinst

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -e
+
+#DEBHELPER#
+
+PACKAGE_NAME="schaufel"
+PACKAGE_USER="schaufel"
+PACKAGE_GROUP="schaufel"
+CONF_DIR="/etc/${PACKAGE_NAME}"
+WORK_DIR="/var/lib/${PACKAGE_NAME}"
+LOG_DIR="/var/log/${PACKAGE_NAME}"
+
+## Create group if not exist
+if ! getent group | grep --quiet "^${PACKAGE_GROUP}:" ; then
+    echo -n "Adding system group ${PACKAGE_GROUP} .."
+    addgroup --quiet --system ${PACKAGE_GROUP} 2>/dev/null || true
+    echo ".. done"
+fi
+
+## Create empty home directory if not exist
+test -d ${WORK_DIR} || mkdir -p ${WORK_DIR}
+
+## Create user if not exist
+if ! getent passwd | grep --quiet "^${PACKAGE_USER}:" ; then
+    echo -n "Adding system user ${PACKAGE_USER} .."
+    adduser --quiet --system \
+        --home ${WORK_DIR} \
```

Comment:
I think we don't need a home dir :rofl: on the gentoo version installation, schaufel user created with home dir pointing to `/dev/null`,